db/migrate/20170822181831_change_conditions.rb

```diff
@@ -0,0 +1,30 @@
+class ChangeConditions < ActiveRecord::Migration[5.1]
+  def change
+    change_table(:conditions) do |t|
+      t.remove :max_dew_point_f
+      t.remove :mean_dew_point_f
+      t.remove :min_dew_point_f
+      t.remove :max_humidity
+      t.remove :min_humidity
+      t.remove :max_sea_level_pressure_inches
+      t.remove :mean_sea_level_pressure_inches
+      t.remove :min_sea_level_pressure_inches
+      t.remove :max_visibility_miles
+      t.remove :min_visibility_miles
+      t.remove :max_wind_Speed_mph
+      t.remove :max_gust_speed_mph
+      t.remove :cloud_cover
+      t.remove :events
+      t.remove :wind_dir_degrees
+  end
+
```

Comment:
I'm assuming this is all just from dropping and migrating. But cool, looks good.